### ISSUE TITLE
local data = DataLoader.new(opt.nThreads, opt)  demo.lua error

### ISSUE THREAD

**Issue by anthonyyuan** [rhgao/on-demand-learning#2]
Starting donkey with id: 1 seed: 334
table: 0x16a75dd0
Creating train metadata
table: 0x16af0f60
running "find" on each class directory, and concatenate all those filenames into a single file containing all image paths for a given class
/tmp/lua_w3KL36: line 1: gfind: command not found
now combine all the files to a single large file
load the large concatenated list of sample paths to self.imagePath
/Users/anthonyyuan/torch/install/bin/luajit: ...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:183: [thread 1 callback] ...mand-learning-master/pixelInterpolation/data/dataset.lua:202: Could not find any image file in the given input paths
stack traceback:
	[C]: in function 'assert'
	...mand-learning-master/pixelInterpolation/data/dataset.lua:202: in function '__init'
	...s/anthonyyuan/torch/install/share/lua/5.1/torch/init.lua:91: in function <...s/anthonyyuan/torch/install/share/lua/5.1/torch/init.lua:87>
	[C]: in function 'dataLoader'
	...earning-master/pixelInterpolation/data/donkey_folder.lua:99: in main chunk
	[C]: in function 'dofile'
	...-demand-learning-master/pixelInterpolation/data/data.lua:38: in function <...-demand-learning-master/pixelInterpolation/data/data.lua:28>
	[C]: in function 'xpcall'
	...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:234: in function 'callback'
	...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:65: in function <...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:41>
	[C]: in function 'pcall'
	...nthonyyuan/torch/install/share/lua/5.1/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:13: in main chunk
stack traceback:
	[C]: in function 'error'
	...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:183: in function 'dojob'
	...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:264: in function 'synchronize'
	...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:142: in function 'specific'
	...honyyuan/torch/install/share/lua/5.1/threads/threads.lua:125: in function 'Threads'
	...-demand-learning-master/pixelInterpolation/data/data.lua:26: in function 'new'
	demo.lua:63: in main chunk
	[C]: in function 'dofile'
	...yuan/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: at 0x0107e58bc0
